config/environments/development.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.configure do
   # Verifies that versions and hashed value of the package contents in the project's package.json
-  config.webpacker.check_yarn_integrity = true
```

Comment:
I think we just need to make this line true again and I could approve. Can you manually change this in the PR?

Reply:
I added it manually below, that should work if you approve now?